package.json

```diff
@@ -353,5 +356,21 @@
 	"volta": {
 		"node": "14.18.2",
 		"npm": "6.14.15"
+	},
+	"nyc": {
+		"include": [
+			"app/**/*.js",
```

Comment:
Isn't better to include all the folders? Like `**/*.js`?

Reply:
I tried to run the coverage with `**/*.(js|ts)` but it didn't work